<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>202</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="mainLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>184</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>200</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="btnClear">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: red;
color: white;</string>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btnPlus">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btnMinus">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="btnDelete">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>‚Üê</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="numpad1">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="numpad2">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="numpad3">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="btnMultiply">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="numpad4">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="numpad5">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="numpad6">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="btnDivide">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="numpad7">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="numpad8">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="numpad9">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" rowspan="2">
     <widget class="QPushButton" name="btnEqual">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>95</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>95</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(175, 175, 175);
color:rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="numpad0">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="btnLbracket">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>(</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="btnRbracket">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
